### ISSUE TITLE
'FIRInstanceID.h' file not found

### ISSUE THREAD

**Issue by daidongon** [firebase/firebase-ios-sdk#3796]
### [REQUIRED] Step 2: Describe your environment

  * Xcode version: 10.2.1
  * Firebase SDK version: 6.8.0
  * Firebase Component: InstanceID (4.2.4), RemoteConfig (4.4.0)
  * Component version: _____
  * Firebase managed by CocoaPods ( 1.7.5 )

### [REQUIRED] Step 3: Describe the problem
Build Failued

`localPath/Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Sources/RCNFetch.m:21:9:` In file included from `localPath/Pods/FirebaseRemoteConfig/FirebaseRemoteConfig/Sources/RCNFetch.m:21:`

<img width="1124" alt="FIRInstanceID_Private_h_と_New_Issue_·_firebase_firebase-ios-sdk" src="https://user-images.githubusercontent.com/25152049/64469790-327ce180-d173-11e9-8a2b-3fb8f5b3eade.png">

**Comment by morganchen12** [comment]
Normally this is something that CocoaPods takes care of. Can you try running `pod deintegrate` and `pod install`? If this doesn't solve the issue, can you share your project?

**Comment by daidongon** [comment]
Thank you for the reply.

> Can you try running pod deintegrate and pod install?

I tried it but it didn't resolve.

## Podfile ( partial )

```ruby
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, '11.0'
install! 'cocoapods', :generate_multiple_pod_projects => true, :incremental_installation => true

use_frameworks!

target 'target' do
  pod 'Firebase/Messaging', :inhibit_warnings => false # Silent warning
  pod 'Firebase/Analytics'
  pod 'Firebase/DynamicLinks'
  pod 'Firebase/RemoteConfig'
  pod 'Firebase/Performance'

  target 'targetTests' do
    inherit! :complete
  end
end
```

## Podfile.lock ( partial )

```
  - Firebase/Analytics (6.8.0):
    - Firebase/Core
  - Firebase/Core (6.8.0):
    - Firebase/CoreOnly
    - FirebaseAnalytics (= 6.1.1)
  - Firebase/CoreOnly (6.8.0):
    - FirebaseCore (= 6.2.2)
  - Firebase/DynamicLinks (6.8.0):
    - Firebase/CoreOnly
    - FirebaseDynamicLinks (~> 4.0.4)
  - Firebase/Messaging (6.8.0):
    - Firebase/CoreOnly
    - FirebaseMessaging (~> 4.1.4)
  - Firebase/Performance (6.8.0):
    - Firebase/CoreOnly
    - FirebasePerformance (~> 3.1.3)
  - Firebase/RemoteConfig (6.8.0):
    - Firebase/CoreOnly
    - FirebaseRemoteConfig (~> 4.4.0)
  - FirebaseABTesting (3.1.1):
    - FirebaseAnalyticsInterop (~> 1.3)
    - FirebaseCore (~> 6.1)
    - Protobuf (~> 3.8)
  - FirebaseAnalytics (6.1.1):
    - FirebaseCore (~> 6.2)
    - FirebaseInstanceID (~> 4.2)
    - GoogleAppMeasurement (= 6.1.1)
    - GoogleUtilities/AppDelegateSwizzler (~> 6.0)
    - GoogleUtilities/MethodSwizzler (~> 6.0)
    - GoogleUtilities/Network (~> 6.0)
    - "GoogleUtilities/NSData+zlib (~> 6.0)"
    - nanopb (~> 0.3)
  - FirebaseAnalyticsInterop (1.4.0)
  - FirebaseCore (6.2.2):
    - FirebaseCoreDiagnostics (~> 1.0)
    - FirebaseCoreDiagnosticsInterop (~> 1.0)
    - GoogleUtilities/Environment (~> 6.2)
    - GoogleUtilities/Logger (~> 6.2)
  - FirebaseCoreDiagnostics (1.0.1):
    - FirebaseCoreDiagnosticsInterop (~> 1.0)
    - GoogleDataTransportCCTSupport (~> 1.0)
    - GoogleUtilities/Environment (~> 6.2)
    - GoogleUtilities/Logger (~> 6.2)
  - FirebaseCoreDiagnosticsInterop (1.0.0)
  - FirebaseDynamicLinks (4.0.4):
    - FirebaseAnalyticsInterop (~> 1.3)
    - FirebaseCore (~> 6.2)
  - FirebaseInstanceID (4.2.4):
    - FirebaseCore (~> 6.0)
    - GoogleUtilities/Environment (~> 6.0)
    - GoogleUtilities/UserDefaults (~> 6.0)
  - FirebaseMessaging (4.1.4):
    - FirebaseAnalyticsInterop (~> 1.3)
    - FirebaseCore (~> 6.2)
    - FirebaseInstanceID (~> 4.1)
    - GoogleUtilities/AppDelegateSwizzler (~> 6.2)
    - GoogleUtilities/Environment (~> 6.2)
    - GoogleUtilities/Reachability (~> 6.2)
    - GoogleUtilities/UserDefaults (~> 6.2)
    - Protobuf (~> 3.1)
  - FirebasePerformance (3.1.3):
    - FirebaseCore (~> 6.2)
    - FirebaseInstanceID (~> 4.2)
    - FirebaseRemoteConfig (~> 4.4)
    - GoogleToolboxForMac/Logger (~> 2.1)
    - "GoogleToolboxForMac/NSData+zlib (~> 2.1)"
    - GoogleUtilities/Environment (~> 6.2)
    - GoogleUtilities/ISASwizzler (~> 6.2)
    - GoogleUtilities/MethodSwizzler (~> 6.2)
    - GTMSessionFetcher/Core (~> 1.1)
    - Protobuf (~> 3.9)
  - FirebaseRemoteConfig (4.4.0):
    - FirebaseABTesting (~> 3.1)
    - FirebaseAnalyticsInterop (~> 1.4)
    - FirebaseCore (~> 6.2)
    - FirebaseInstanceID (~> 4.2)
    - GoogleUtilities/Environment (~> 6.2)
    - "GoogleUtilities/NSData+zlib (~> 6.2)"
    - Protobuf (~> 3.9)
  - GoogleAppMeasurement (6.1.1):
    - GoogleUtilities/AppDelegateSwizzler (~> 6.0)
    - GoogleUtilities/MethodSwizzler (~> 6.0)
    - GoogleUtilities/Network (~> 6.0)
    - "GoogleUtilities/NSData+zlib (~> 6.0)"
    - nanopb (~> 0.3)
  - GoogleDataTransport (1.2.0)
  - GoogleDataTransportCCTSupport (1.0.3):
    - GoogleDataTransport (~> 1.2)
    - nanopb
```

**Comment by daidongon** [comment]
The application was successfully built by changing to the following settings.

```ruby
install! 'cocoapods', :generate_multiple_pod_projects => false, :incremental_installation => false
```

**Comment by paulb777** [comment]
@daidongon Thanks for the report and narrowing it down to the CocoaPods configuration.  We're working on a fix at #3803
